intervaltree/interval.py

```diff
@@ -53,6 +53,31 @@ def overlaps(self, begin, end=None):
         except:
             return self.contains_point(begin)
 
+    def overlap_size(self, begin, end=None):
+        """
+        Return the overlap size between two intervals or a point
+        :param begin: beginning point of the range, or the point, or an Interval
+        :param end: end point of the range. Optional if not testing ranges.
+        :return: Return the overlap size, None if not overlap is found
+        :rtype: int
+        """
+        overlaps = self.overlaps(begin, end)
+        if not overlaps:
+            return None
```

Comment:
Could we return 0 here? This way the caller doesn't have to do as many checks. Also, 0 is a value that is correct and still makes sense.

Reply:
agree